docs/manuals/source/DeveloperGuide.rst

```diff
@@ -5,6 +5,7 @@ Developer Guide
 
    DeveloperGuide/generaldevel.rst
    DeveloperGuide/BuildAndTestBareos.rst
+   DeveloperGuide/release.rst
```

Comment:
```suggestion
   DeveloperGuide/ReleasingBareos.rst
```

See https://docs.bareos.org/master/DocumentationStyleGuide/BareosSpecificFormatting/DocumentFileStructure.html.
This is used throughout the document, except the DeveloperGuide. There it is only used for new documents.